src/main/webapp/script.js

```diff
@@ -62,6 +40,37 @@ function toggleAQLayer() {
     aqLayer.setMap(aqLayer.getMap() ? null : map);
 }
 
+function loadAQVisualisationData() {
```

Comment:
Good practice to name functions for the thing they do (verb based naming); in this case 'load', but a better verb for this function would be 'fetch', or since it doesn't return anything, perhaps 'populate'. Mismatched verbs are less readable (because they set the wrong expectations), but more common if the writer isn't as well versed on the "typical" use of the verbs.

Reply:
What are the usual connotations behind load?